### PR TITLE
New 'create new user' tutorial

### PR DESCRIPTION
**Pull request by dougmanis**
added new 'create user' tutorial

Notes: 
* this pull request supersedes the existing request #25  
* Request #25 had some branching and merging issues so I cherry picked the changes to a new branch and new pull request 

changes:
* added new 'add-a-user_dmanis.md' tutorial file
* added link to the new tutorial from the existing docs index topic
* supporting assignment 6.2: 'Write a first use or a tutorial topic'

Applied edits from 6.2 code review

Changes:
* Expanded intro text
* Added an 'Overview' section
* Applied minor edit suggestions
* Added 'What's next?' section
* Fixed minor Markdown formatting warnings throughout
* Fixed anchor link to 'fetch user details'

### CODE REVIEW

**Comment by dougmanis** [comment]
Notes:
* This pull request supersedes the existing request https://github.com/UWC2-APIDOC/to-do-service-au24/pull/25 (https://github.com/UWC2-APIDOC/to-do-service-au24/pull/25) 
* Request https://github.com/UWC2-APIDOC/to-do-service-au24/pull/25 had some branching and merging issues so I cherry picked the changes to a new branch and new pull request

**Comment by rbwatson** [comment]
Rebased PR to current `main` branch for merge.

File: docs/index.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ First, do this tutorial to set up your development system for these tutorials. Y
 After your system is ready, [tutorials](./tutorials.md) such as these show you how to perform common tasks.
 
 * [Enroll a new user](tutorials/enroll-a-new-user.md)
+* [First Use (new user) DMANIS](tutorials/add-a-user_dmanis.md)
```
Comment (rbwatson) [inline docs/index.md:47]: Moved link to be with other tutorial links.

**Review by rbwatson — APPROVED** [review]
I removed the change to .gitignore. Adding the database file to the ignore list would remove it from the repo for all practical purposes.

LGTM

**Comment by rbwatson** [comment]
This PR was reopened in #58, where it was merged.

**Comment by dougmanis** [comment]
> I removed the change to .gitignore. Adding the database file to the ignore list would remove it from the repo for all practical purposes.
> 
> LGTM

Thank you for the catch! I did not think that through.

**Pull request merged**